atlp-rwanda/atlp-devpulse-fn
# Circleci project setup

What does this PR do?

This PR indicates project folder structure and folders/files naming conversion.
It can be configured to run very complex pipelines efficiently with sophisticated caching, docker layer caching, and resource classes for running on faster machines.

What to expect?

It believes in the configuration as code. Consequently, the entire delivery process from build to deployment is orchestrated through a single file called config.yml.

How to test this PR?

Clone this [repo](https://github.com/atlp-rwanda/atlp-devpulse-fn/circleci-project-setup)
run the command git checkout Project-setup (after being merged you will no longer need this command)
run the command `npm install`
run the command `npm run dev` to start the server
